core/services/vrf/delegate.go

```diff
@@ -426,6 +438,18 @@ func (lsn *listener) handleLog(lb log.Broadcast, minConfs uint32) {
 	lsn.reqsMu.Unlock()
 }
 
+func (lsn *listener) shouldProcessLog(lb log.Broadcast) bool {
+	ctx, cancel := postgres.DefaultQueryCtx()
+	defer cancel()
+	consumed, err := lsn.logBroadcaster.WasAlreadyConsumed(lsn.db.WithContext(ctx), lb)
+	if err != nil {
```

Comment:
would it be possible to match on part of the error message as well?

Reply:
sorry what do you mean by match?

Reply:
oh, just `strings.Contains(...)` or similar - to keep in the codebase the exact error that happens in this case, if possible

Reply:
there's probably a couple different db read errors it could be, but seems safer to just log and return on all errors vs trying to match them all

Reply:
ok - yeah true